### ISSUE TITLE
Keyboard-Interactive Authentication

### ISSUE THREAD

**Issue by vampire337** [vmware/photon#914]
After running tdnf distro-sync on a newly-deployed Photon 3 VM as well as on all my existing Photon 3 VMs, I'm now experiencing a minor/annoyance issue connecting via SSH.  I use KiTTY for SSH, and I use it to store the passwords for administering my local network VMs.  Usually I can just open the link to whichever server I'm connecting to and it logs me in (as root in this case).  Starting last evening after updates, I am now presented with some slightly different prompts which include the text "Using keyboard-interactive authentication", and it requires me to hit enter to accept the password.  Which is a little annoying but not a deal-breaker.

Now if my connection is interrupted and my client must relogin (such as in the event of switching from WiFi to hard-wired LAN, or I rebooted the VM) I get the same prompt, but this time pressing enter does nothing, and ultimately I have to restart the client.  Some googling suggests it may be a configuration on the server side, but since it's new as of one of the recent updates I thought it seemed like something I should post here.  Not sure if the switch from password authentication to keyboard-interactive authentication might have been intentional or was just an oversight, but I'm looking for whether others are seeing this, and also I'd like for it to go back to what had been working more smoothly.

**Comment by vampire337** [comment]
I just noted one VM which did not get updated, which doesn't enforce keyboard-interactive authentication yet.  Tdnf reports the following:
> 2 Security notice(s)
> Run 'tdnf updateinfo info' to see the details.
> root@plexmovies [ ~ ]# tdnf updateinfo info
>        Name : bzip2-libs-1.0.6-10.ph3.x86_64.rpm
>   Update ID : patch:PHSA-2019-3.0-0023
>        Type : Security
>     Updated : Thu Jul  4 00:01:34 2019
> Needs Reboot: 0
> Description : Security fixes for {'CVE-2019-12900'}
>        Name : bzip2-1.0.6-10.ph3.x86_64.rpm
>   Update ID : patch:PHSA-2019-3.0-0023
>        Type : Security
>     Updated : Thu Jul  4 00:01:34 2019
> Needs Reboot: 0
> Description : Security fixes for {'CVE-2019-12900'}

I noted that 2019-3.0-0023 hasn't yet been updated on the wiki (https://github.com/vmware/photon/wiki/Security-Advisories-3), but this at least helps point out my culprit.

**Comment by vampire337** [comment]
Okay, I think this bug report can be disregarded.  I have confirmed that my issue is/was client-related.

If anyone else finds this through a search, check your SSH client version.  In my case, I apparently had updated KiTTY to a 0.71 version; when connecting with an older 0.68 version I'm able to log in as normal.  I confirmed KiTTY versions 0.71.0.2 through 0.71.0.5 (most recent as of this post) all display this issue, and I also confirmed that version 0.70.0.10 logs into Photon 3.0 just fine with all the updates.  I have not tested whether similar versions of PuTTY are affected, although it's worth noting that the KiTTY developer mentions on their website that maintaining the software through the jump from PuTTY 0.70 to 0.71 has been problematic, so I wouldn't be surprised if that's my culprit.  I have reached out to the developer to report the issue.
